lib/api/route/channel.dart

```diff
@@ -29,3 +29,23 @@ class SubscribeToChannelsResult {
   Map<String, dynamic> toJson() => _$SubscribeToChannelsResultToJson(this);
 }
 
+Future<UnsubscribeFromChannelsResult> unsubscribeFromChannels(ApiConnection connection, List<ZulipStream> streams) {
```

Comment:
Let's link to the relevant API documentation here.

Reply:
We should use dart-doc for the API documentation link like we do for other routes.